packages/vite-plugin-cloudflare/playground/node-compat/__tests__/worker-warnings/warnings.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { readFileSync, writeFileSync } from "node:fs";
+import { join } from "node:path";
+import dedent from "ts-dedent";
+import { expect, test, vi } from "vitest";
+import { serverLogs } from "../../../__test-utils__";
+
+test("should display warnings if nodejs_compat is missing", async ({
+	onTestFinished,
+}) => {
+	const wranglerConfigPath = join(
+		__dirname,
+		"../../worker-warnings/wrangler.toml"
+	);
+	const originalWranglerConfig = readFileSync(wranglerConfigPath, "utf8");
+	onTestFinished(() => {
+		writeFileSync(wranglerConfigPath, originalWranglerConfig);
+	});
```

Comment:
why is this needed? is something modifying the config file?

Reply:
Ah! It is not any more. Previously I was modifying the files during the test and needed to return them to their normal state.

Will remove!